Stub maker commands: missing command name in error

Hi,

When running e.g. `symfony console make:factory`, the error message is missing the command name:

```
 [ERROR] To run "" you need the "MakerBundle" which is currently not installed.

         Try running "composer require symfony/maker-bundle --dev".
```

It's because the stub command uses `static::$defaultName` which is not defined anymore.

--- a/src/Bundle/Command/StubCommand.php
+++ b/src/Bundle/Command/StubCommand.php
@@ -22,10 +22,10 @@ abstract class StubCommand extends Command
     {
         (new SymfonyStyle($input, $output))
             ->error(
-                \sprintf("To run \"%s\" you need the \"%s\" which is currently not installed.\n\nTry running \"composer require %s\".", static::$defaultName, 'MakerBundle', 'symfony/maker-bundle --dev')
+                \sprintf("To run \"%s\" you need the \"MakerBundle\" which is currently not installed.\n\nTry running \"composer require symfony/maker-bundle --dev\".", static::getDefaultName())
             )
         ;
 
-        return Command::SUCCESS;
+        return Command::FAILURE;
     }
 }

